stages/org.osbuild.oscap.autotailor

```diff
@@ -15,15 +15,19 @@ def main(tree, options):
     new_profile = config.get("new_profile")
```

Comment:
We can't rename it to `tailored_profile`, can we?

Reply:
Theoretically we could and try make it backwards compatible